jiant/tasks/tasks.py

```diff
@@ -1545,15 +1548,25 @@ class RTESuperGLUETask(RTETask):
     def load_data(self):
         """ Process the datasets located at path. """
         targ_map = {"not_entailment": 0, True: 1, False: 0, "entailment": 1}
-        self.train_data_text = load_pair_nli_jsonl(
```

Comment:
Also, while you're at it, can you change the superglue-diagnosticheader so that the relative path is RTE/diagnostics?

Reply:
I'll address in the other diagnostics PR